cardano-api/src/Cardano/Api/Block.hs

```diff
@@ -250,7 +250,7 @@ getBlockHeader (ByronBlock block)
 --
 
 data ChainPoint = ChainPointAtGenesis
-                | ChainPoint !SlotNo !(Hash BlockHeader)
+                | ChainPoint !SlotNo !(Hash BlockHeader) (Maybe EpochNo)
```

Comment:
Oh, no we cannot change this. A point is fundamental. This is what it is, no more no less.

In the CLI we can pair up a point with an epoch number, but we cannot change the point type.